sql/moz-fx-data-shared-prod/mozilla_vpn_derived/channel_group_proportions_live/view.sql

```diff
@@ -5,6 +5,7 @@ WITH stage_1 AS (
   SELECT
     event_date AS subscription_start_date,
     country_name,
+    country,
```

Comment:
Select column order needs to match what's in `schema.yaml`.  I think the order in `schema.yaml` makes more sense, so I suggest changing this.
```suggestion
    country,
    country_name,
```